src/templates/releaseNotePage.js

```diff
@@ -29,7 +29,7 @@ class ReleaseNoteTemplate extends React.Component {
       <Layout containerWidth={containerWidth} excludeSearch footerBorder>
         <SEO
           title={node.frontmatter.title}
-          description={node.frontmatter.description || node.excerpt}
+          description="A summary of changes to the Pantheon Platform"
```

Comment:
@jazzsequence can you try to get excerpts working for release note meta descriptions? Switching assignee of the PR to you for next steps